sqsworkers/crew.py

```diff
@@ -197,12 +273,12 @@ def custom_exception_handler(self, e, message):
             'streamhub_event_type': failed_streamhub_event_type,
             'streamhub_event_schema': failed_streamhub_event_schema
         }
-        self.logger.error('There was an error processing the message %s' % e)
```

Comment:
why?

Reply:
Will revert back these random attempts to replace old way with new